README.md

```diff
@@ -1 +1,4 @@
+![smelt-modding.github.io](roundedlogo.jpg)
+### Smelt - A file server for SBPM packages and installation
```

Comment:
The tagline for this repo is "File server for SBPM packages and installation" but the tagline of SBPM itself is "Smelt: Batch Package Manager" (also what "SBPM" stands for). Could you change this? Thanks. Otherwise, good work, I guess

Reply:
Okay 👍

Reply:
Done :D